probe-rs/src/debug/debug_info.rs

```diff
@@ -1120,6 +1121,35 @@ impl DebugInfo {
     }
 }
 
+/// This is a fuzzy 'equality' check for path names.
+/// If both paths are absolute, then it will use the standard `Path::eq` method.
+/// If the secondary path is relative, then it will succeeed if the primary ends with the compared path,
+/// e.g. `/home/user/project/src/main.rs` ends with `src/main.rs` and will return true.
+pub(crate) fn path_fuzzy_eq(primary_path: &Path, secondary_path: &Path) -> bool {
```

Comment:
Shouldn't this just be an https://doc.rust-lang.org/std/path/struct.Path.html#method.ends_with?

Reply:
No, because `ends_with()` doesn't take into account the leading `.` paths.

Reply:
Yes but you can remove the leading `.` and then use ends with instead of own complicated logic :)